solutions/bio2/TickTock/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { SCodeBlock, SText, STitle } from "components";
+import { brace } from "solutions/util";
+
+export const TickTock = () => {
+  return (
+    <>
+      <STitle>Idea</STitle>
+      <STitle>Abridged Statement</STitle>
+      <SText>
+      You have $n$ clocks, each of which are audible from time $f$ to $l$ inclusive and make a tick sound when they are audible and the sum of the digits of the time they display is even.
+      </SText>
+      <SText>
+      The input given is the integer $n$ and then $n$ lines of two integers $f_i$ and $l_i$, corresponding to the $f$ and $l$ values of the audible period of the $i$th clock.
```

Comment:
Is there a way to make the "th" a superscript?

Reply:
Yes, there is.